app/notification/routes.py

```diff
@@ -0,0 +1,24 @@
+from app.config import API_KEY
+from app.config import TEMPLATE_ID
+from flask import Blueprint
+from notifications_python_client.notifications import NotificationsAPIClient
+
+notification_bp = Blueprint(
+    "notification_bp",
+    __name__,
+    url_prefix="/send",
+    template_folder="templates",
+)
+
+notifications_client = NotificationsAPIClient(API_KEY)
+
+
+@notification_bp.route("/", methods=["GET", "POST"])
+def send_notification():
+    email_address = "steve.walton@communities.gov.uk"
```

Comment:
:wrench: Replace email  with env variable

Reply:
Waiting on Cary to add email to env hence failing. Updated locally, works fine

Reply:
I just don't want Steve's email to get spammed by gitihub bots haha

Reply:
email added to gov uk secrets